server/views/pages/overview/partials/overviewTabActionPlanColumn.njk

```diff
@@ -0,0 +1,101 @@
+{% if not actionPlan.problemRetrievingData %}
```

Comment:
This new partial is essentially what was originally in `overviewTabContent.njk` - I've pulled it all out into a partial in its own right as I suspect the work that @chrimesdev is doing re: adding the Functional Skills to the right hand column of the Overview tab would otherwise start making that template file absolutely massive!